lib/bbbevents/attendee.rb

```diff
@@ -57,9 +58,16 @@ def csv_row
       ].map(&:to_s)
     end
 
+    def to_hash
```

Comment:
I'd prefer that you use the `to_h` name for this method - `to_hash` has special meaning in ruby (it's used for implicit conversions)

Reply:
Renamed to `to_h`